services/ui-src/src/page/waiver-renewal/WaiverRenewalForm.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from "react";
+import OneMACForm from "../OneMACForm";
+import { defaultOneMACFormConfig, OneMACFormConfig } from "../../libs/formLib";
+import { ROUTES, ONEMAC_ROUTES, waiverRenewal } from "cmscommonlib";
+
+const waiverRenewalIdFormat: string = "SS-####.R##.00 or SS-#####.R##.00";
+
+export const waiverRenewalFormInfo: OneMACFormConfig = {
+  ...waiverRenewal,
+  ...defaultOneMACFormConfig,
```

Comment:
do we want to switch this around to have the default spread first in case anything in the more specific config needs to overwrite something in the default?

Reply:
yes, good catch!